internal/service/ipam/model_ipamsvc_address_block.go

```diff
@@ -0,0 +1,446 @@
+	"asm_config":                    types.ObjectType{AttrTypes: IpamsvcASMConfigAttrTypes},
+	"asm_scope_flag":                types.Int64Type,
+	"cidr":                          types.Int64Type,
+	"comment":                       types.StringType,
+	"created_at":                    timetypes.RFC3339Type{},
+	"ddns_client_update":            types.StringType,
+	"ddns_conflict_resolution_mode": types.StringType,
+	"ddns_domain":                   types.StringType,
+	"ddns_generate_name":            types.BoolType,
+	"ddns_generated_prefix":         types.StringType,
+	"ddns_send_updates":             types.BoolType,
+	"ddns_ttl_percent":              types.Float64Type,
+	"ddns_update_on_renew":          types.BoolType,
+	"ddns_use_conflict_resolution":  types.BoolType,
+	"dhcp_config":                   types.ObjectType{AttrTypes: IpamsvcDHCPConfigAttrTypes},
+	"dhcp_options":                  types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcOptionItemAttrTypes}},
+	"dhcp_utilization":              types.ObjectType{AttrTypes: IpamsvcDHCPUtilizationAttrTypes},
+	"discovery_attrs":               types.MapType{ElemType: types.StringType},
+	"discovery_metadata":            types.MapType{ElemType: types.StringType},
+	"header_option_filename":        types.StringType,
+	"header_option_server_address":  types.StringType,
+	"header_option_server_name":     types.StringType,
+	"hostname_rewrite_char":         types.StringType,
+	"hostname_rewrite_enabled":      types.BoolType,
+	"hostname_rewrite_regex":        types.StringType,
+	"id":                            types.StringType,
+	"inheritance_parent":            types.StringType,
+	"inheritance_sources":           types.ObjectType{AttrTypes: IpamsvcDHCPInheritanceAttrTypes},
+	"name":                          types.StringType,
+	"parent":                        types.StringType,
+	"protocol":                      types.StringType,
+	"space":                         types.StringType,
+	"tags":                          types.MapType{ElemType: types.StringType},
+	"threshold":                     types.ObjectType{AttrTypes: IpamsvcUtilizationThresholdAttrTypes},
+	"updated_at":                    timetypes.RFC3339Type{},
+	"usage":                         types.ListType{ElemType: types.StringType},
+	"utilization":                   types.ObjectType{AttrTypes: IpamsvcUtilizationAttrTypes},
+	"utilization_v6":                types.ObjectType{AttrTypes: IpamsvcUtilizationV6AttrTypes},
+}
+
+var IpamsvcAddressBlockResourceSchemaAttributes = map[string]schema.Attribute{
+	"address": schema.StringAttribute{
+		Required: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
+		MarkdownDescription: "The address field in form “a.b.c.d/n” where the “/n” may be omitted. In this case, the CIDR value must be defined in the _cidr_ field. When reading, the _address_ field is always in the form “a.b.c.d”.",
```

Comment:
We shouldn't allow the `/n` notation for the address in the terraform resource. It creates a non empty plan after apply. But we will have to do validation. for it. Not for this PR, but it applies to subnet as well